libnavigation-core/src/main/java/com/mapbox/navigation/core/navigator/LocationEx.kt

```diff
@@ -116,12 +117,21 @@ internal fun Bundle.toMap(): FixLocationExtras {
     while (iterator.hasNext()) {
         val key = iterator.next()
         when (val value = this.get(key)) {
+            is Boolean -> map[key] = Value(value)
+            is Byte -> map[key] = Value(value.toLong())
+            is Char -> map[key] = Value(value.toString())
             is Double -> map[key] = Value(value)
+            is Float -> map[key] = Value(value.toDouble())
+            is Int -> map[key] = Value(value.toLong())
             is Long -> map[key] = Value(value)
-            is Boolean -> map[key] = Value(value)
+            is Short -> map[key] = Value(value.toLong())
             is String -> map[key] = Value(value)
             else -> {
-                // do nothing
+                Log.w(
```

Comment:
As far as I remember we recently switched to using common logger everywhere `com.mapbox.navigation.utils.internal.logW`. @Guardiola31337, @LukasPaczos, can you please confirm 🙂

Reply:
Should it be `assert` instead btw?

Reply:
Do you mean this [assert](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/assert.html)? Like to cash on internal builds but do nothing on customer's build?

I like the idea, but we currently don't use asserts. I would leave just `logW` 🙂

Reply:
> we recently switched to using common logger everywhere

That's right, let's use `com.mapbox.navigation.utils.internal.logW`.